test/BaselineOutput/SingleDebug/Text/whitened.tsv

```diff
@@ -0,0 +1,8 @@
+#@ TextLoader{
```

Comment:
>TextLoader [](start = 3, length = 10)

Let's put this file into Common/Text/ instead of two versions of same file in Single<Version>/Text/